src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/config/SchedulerConfig.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.config;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.scheduling.annotation.SchedulingConfigurer;
+import org.springframework.scheduling.concurrent.ThreadPoolTaskScheduler;
+import org.springframework.scheduling.config.ScheduledTaskRegistrar;
+
+@Configuration
+public class SchedulerConfig implements SchedulingConfigurer {
```

Comment:
`@EnableScheduling` should go together with `SchedulingConfigurer` implementation 😉